twine/commands/upload.py

```diff
@@ -140,6 +140,13 @@ def upload(upload_settings: settings.Settings, dists: List[str]) -> None:
 
         resp = repository.upload(package)
 
+        if upload_settings.verbose:
+            logger.info(
+                f"Received {resp.status_code} response from {resp.url}: {resp.reason}"
+            )
+            if resp.text:
+                logger.info(f"Response text:\n{resp.text}")
```

Comment:
Probably makes sense to use https://toolbelt.readthedocs.io/en/latest/dumputils.html for this instead.

Reply:
Done in https://github.com/pypa/twine/pull/859/commits/b458254ee7c13e576982abcff34756da7c7103e0.